template/README.md

```diff
@@ -511,6 +512,39 @@ To override this, specify the `homepage` in your `package.json`, for example:
 
 This will let Create React App correctly infer the root path to use in the generated HTML file.
 
+## Netlify
+
+**To do a manual deploy to Netlify's CDN:**
+
+```sh
+npm install netlify-cli
+netlify deploy
+```
+
+Choose `build` as the path to deploy.
+
+**To setup continuous delivery**
+
+With this setup Netlify will build and deploy when you push to git or open a pull request:
+
+1. [Start a new netlify project](https://app.netlify.com/signup)
+2. Pick your Git hosting service and select your create-react-app repository
+3. Click `Build your site`
+
+**Support for client site routing:**
+
+To support `pushState` add a `src/_redirects` file with the following rewrite rule:
+
+```
+/*  /index.html  200
+```
+
+Then add this import statement to `src/index.js`:
+
+```js
+import "file?name=_redirects!./_redirects";
```

Comment:
But now it’s confusing because we already create a folder called `static` in `build` for static assets. If we allow `src/static` people will expect things there to be in `build/static/*`, not in `build`. But we want to support the “root files” (at least that’s what this PR requires).